src/index.html

```diff
@@ -2,11 +2,431 @@
+            alt="logo"
+          >
+        </a>
+      </div>
+      <div class="nav-container">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#"
+              >Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+                data-qa="nav-hover"
+              >Laptops &#x26; computers</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="cards-container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="product-image">
+            <img
+              class="image"
+              src="./images/imac.jpeg"
+              alt="product-image"
+            >
+          </div>
+
+          <div class="product-description">
+            <a
+              class="product-description__link"
+              href="#"
+            >
+              APPLE A1419 iMac 27" Retina 5K
+              Monoblock (MNED2UA/A)
+            </a>
+          </div>
+
+          <div class="product-code">Product code: 195434</div>
+          <div class="product-rate">
+            <div class="product-rate__stars active-stars-4">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <div class="product-rate__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product-price">
+            <p class="product-price__price">Price:</p>
+            <p class="product-price__amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            data-qa="card-hover"
+          >
+            <button class="buy-button">Buy</button>
+          </a>
```

Comment:
Button tag is redudant here 
```suggestion
          <a
            href="#"
            data-qa="card-hover"
            class="buy-button"
          >
            Buy
          </a>
```